components/automate-cli/pkg/verifyserver/server/api/v1/externalpostgresql.go

```diff
@@ -0,0 +1,21 @@
+package v1
+
+import (
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/models"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/response"
+	"github.com/gofiber/fiber/v2"
+)
+
+func (h *Handler) CheckExternalPostgresql(c *fiber.Ctx) error {
+	externalPostgresqlRequest := new(models.ExternalPgRequest)
+	if err := c.BodyParser(&externalPostgresqlRequest); err != nil {
```

Comment:
Add a validator to check the received values are non-empty strings

Reply:
done